src/components/Header/Header.tsx

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
```

Comment:
Consider removing the 'eslint-disable jsx-a11y/control-has-associated-label' comment and adding an 'aria-label' attribute to the button element in line 69 to improve accessibility.